workspace_host/interface.sql

```diff
@@ -7,7 +7,7 @@ FROM
   workspaces AS w
   JOIN variants AS v ON (v.workspace_id = w.id)
   JOIN questions AS q ON (q.id = v.question_id)
-  JOIN pl_courses AS c ON (c.id = q.course_id)
+  JOIN pl_courses AS c ON (c.id = v.course_id)
```

Comment:
I think this is good too. AFAICT this is only used to get a course ID that we can label S3 objects with, and we do want them to be associated with the variant's course.